src/pages/SupplierInvoicePage.js

```diff
@@ -225,15 +352,42 @@ export class SupplierInvoicePage extends React.Component {
           icon: 'md-remove-circle',
           isDisabled: transaction.isFinalised,
         };
+      case 'vvm': {
+        if (!isVaccine) return emptycell;
+        return (
+          <MiniToggleBar
+            leftText="PASS"
+            rightText="FAIL"
+            currentState={isVVMPassed}
+            onPress={this.onVvmToggle(transactionBatch)}
+          />
+        );
+      }
+      case 'fridge': {
+        if (!isVaccine) return emptycell;
+        const failed = isVVMPassed === false;
```

Comment:
should we show 'no fridges' <- if no default fridge ?

Reply:
done 0908a8201be02fa258d976dd99b9201dd7e2f1a2

3 states for fridge cell: 'No fridges'/'Discarded' (both using the same style, with a cross and low opacity gray) and pressable 'Fridge X (name)' with a caret